src/Shim.Extensions/Forwards.cs

```diff
@@ -3,20 +3,25 @@
 using System.Runtime.CompilerServices;
 
 [assembly: TypeForwardedTo(typeof(System.Web.Script.AjaxFrameworkAssemblyAttribute))]
+[assembly: TypeForwardedTo(typeof(System.Web.Script.Services.ResponseFormat))]
```

Comment:
these should be added when the API itself is added - there's a test to verify that the APIs are all being forwarded

Reply:
if there are APIs that are in the project but not being forwarded, the test should be updated :)